zipline/testing/fixtures.py

```diff
@@ -614,12 +614,7 @@ def _make_equity_daily_bar_from_minute(cls):
             'high': 'max',
             'low': 'min',
             'close': 'last',
-            # TODO: Change test data so that large minute volumes are not used,
-            # so that 'sum' can be used without going over the uint limit.
-            # When that data is changed, this function can and should be moved
-            # to the `data` module so that loaders and tests can use the same
-            # source from minute logic.
-            'volume': 'last'
```

Comment:
@yankees714 that refactoring has been squashed in, the new function is here https://github.com/quantopian/zipline/pull/1372/files#diff-64eeb1b15a7a2ce1ec06af6c9a2f4b47R24